app/models/broadcast.rb

```diff
@@ -2,7 +2,7 @@ class Broadcast < ApplicationRecord
   VALID_BANNER_STYLES = %w[default brand success warning error].freeze
   resourcify
 
-  has_many :notifications, as: :notifiable, inverse_of: :notifiable
+  has_many :notifications, as: :notifiable, inverse_of: :notifiable, dependent: :destroy
```

Comment:
I read up on this briefly, but this just ensures that associated data (via foreign keys) is destroyed (cleaned up) right? https://stackoverflow.com/a/29560982/77814

Reply:
Exactly!

This means that if we call `broadcast.destroy` all `broadcast.notifications` will be destroyed as well, otherwise they'll be there hanging in the DB and invalid